repos/system_upgrade/common/actors/selinux/selinuxapplycustom/Makefile

```diff
@@ -1,2 +1,5 @@
 install-deps:
-	yum install -y policycoreutils policycoreutils-python
+	yum install -y policycoreutils
+	-yum install -y policycoreutils-python
+	-yum install -y python3-policycoreutils
```

Comment:
@vinzenz just thinking about nitpick, what about change it to:

```
yum install -y policycoreutils-python || yum install -y python3-policycoreutils
```
So python-3policycoreutils is tried to be installed only when policycoreutils-python has not been installed? As well, in this way we are sure that at least on of the really has been installed. In the current way, we could continue even when the expected package has not been installed eg. because of troubles with networking (I expect that we are speaking just about rare corner case, my point mainly about saving another couple of seconds for py2 systems). It's really nitpick, so I am ok if you want to keep it as it is.

Reply:
@vinzenz mybe even better, the whole block to:
```
yum install -y policycoreutils policycoreutils-python || yum install -y policycoreutils python3-policycoreutils
```

That will save even more time :)

Reply:
Well, this is not skipping the next line, it continues with it and if would fail.
'-' just ignores the failure

Reply:
Will apply in #760 as discussed offline